ni_measurementlink_service/measurement/info.py

```diff
@@ -11,7 +11,7 @@ class MeasurementInfo(NamedTuple):
 
     Attributes
     ----------
-        display_name (str): The measurement display name for client to display to user.
+        display_name (str): The measurement display name for clients to display to users.
```

Comment:
Any reason the type was removed/omitted in doc updates about, but (str) is present here?

Reply:
Sphinx can read type hints, so types in docstrings are unnecessary. However, I didn't remove the types from the docstrings I made minor updates to.

Reply:
https://github.com/ni/measurementlink-python/pull/378